.github/workflows/build.yml

```diff
@@ -102,42 +100,43 @@ jobs:
       run: make PLATFORM=x86_64-pc-oslab A=examples/helloworld
     - name: Build httpclient for x86_64-pc-oslab
       continue-on-error: ${{ matrix.rust-toolchain == 'nightly' }}
-      run: make PLATFORM=x86_64-pc-oslab A=examples/httpclient FEATURES=driver-ixgbe
+      run: make PLATFORM=x86_64-pc-oslab A=examples/httpclient FEATURES=page-alloc-4g,driver-ixgbe
```

Comment:
I think it's better to make it as a default feature, 
while left `page-alloc-64g` as a optional one.

Reply:
I thik we still need `page-alloc-256m` for tiny platform. If we left `page-alloc-4g` as default, we can not use `BitAlloc256M` as the feature resolving logic in crate `alloctor`.